packages/nimble-components/src/rich-text-editor/specs/README.md

```diff
@@ -396,6 +405,7 @@ library. For the currently supported features, we will include the following lib
 -   [@tiptap/core](https://www.npmjs.com/package/@tiptap/core)
 -   [@tiptap/pm](https://www.npmjs.com/package/@tiptap/pm)
 -   [@tiptap/starter-kit](https://www.npmjs.com/package/@tiptap/starter-kit)
+-   [@tiptap/extension-character-count](https://www.npmjs.com/package/@tiptap/extension-character-count)
```

Comment:
If the direction of not handling maxlength is agreed then reminder to remove this dep

Reply:
Removed